### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by serhido**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
-    pass
+    result = []
+    for person in people:
+        person = Person(person["name"], person["age"])
+        result.append(person)
+    for partner in people:
+        name = partner["name"]
+        if partner.get("wife"):
+            Person.people[name].wife = Person.people[partner["wife"]]
+            partner["wife"] = Person.people[partner["wife"]]
```
Comment (katevasilenko) [inline app/main.py:19]: Why do you need this line?

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+        elif partner.get("husband"):
+            Person.people[name].husband = Person.people[partner["husband"]]
+            partner["husband"] = Person.people[partner["husband"]]
```
Comment (katevasilenko) [inline app/main.py:22]: The same question. I don’t get it

**Review by katevasilenko — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+        self.people.update({self.name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for person in people:
+        person = Person(person["name"], person["age"])
+        result.append(person)
```
Comment (Vasyl-Smutok) [inline app/main.py:14]: try to use list compression

**Review by Vasyl-Smutok — CHANGES_REQUESTED** [review]

**Review by Vasyl-Smutok — APPROVED** [review]
Good Job 👍

**Pull request closed without merging**